diesel_tests/tests/expressions/mod.rs

```diff
@@ -67,6 +87,51 @@ fn test_count_max() {
     assert_eq!(Ok(None::<i32>), source.first(&connection));
 }
 
+table! {
+    number_arrays (na) {
+        na -> Array<Integer>,
+    }
+}
+
+#[test]
+#[cfg(feature = "postgres")]
+fn test_min_max_of_array() {
+    use self::number_arrays::columns::*;
+    use self::number_arrays::table as number_arrays;
+
+    let connection = connection();
+
+    connection
+        .execute("CREATE TABLE number_arrays (na int[])")
+        .unwrap();
```

Comment:
So I think (?) this is wrong and I should be using a file in in https://github.com/diesel-rs/diesel/tree/master/migrations/postgresql

But what is the format for naming those folders?